app/src/main/java/org/mozilla/fenix/components/TopSiteStorage.kt

```diff
@@ -88,8 +88,15 @@ class TopSiteStorage(private val context: Context) {
         }
     }
 
+    /**
+     * This is for issue https://github.com/mozilla-mobile/fenix/issues/11660. We prefetch the top
+     * sites for startup so that we're sure that we have all the data available as our fragment is
+     * launched. This method doesn't affect performance since the [getTopSites] runs on the a
+     * background thread. ObserveOnce also removes our observer once the value of the
```

Comment:
nit: I don't think the comment about ObserverOnce is necessary – it's kind of redundant to the code.